canonical-web-and-design/jaas-dashboard
# Use event listener hook

## Done

Move complexity around event listeners in useEffects into a custom hook where instantiation and cleanup are abstracted away.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Ensure all tests pass
- Check code
